services/director-v2/.env-devel

```diff
@@ -25,8 +25,6 @@ DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID=1234
 DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME=1234
 DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME=filename
 
-DIRECTOR_V2_PUBLIC_API_BASE_URL=http://127.0.0.1:8006
```

Comment:
make sure that the MR(s) with the removal of this env var is added. you will need 3 for: master, staging and production deployments
I did not see them linked to this PR

Reply:
Yup, I'm preparing them now :)